lib/museum.rb

```diff
@@ -0,0 +1,43 @@
+require 'pry'
+class Museum 
+  attr_reader :name, :exhibits, :patrons
+  def initialize(name)
+    @name = name
+    @exhibits = []
+    @patrons = []
+  end
+
+  def add_exhibit(exhibit)
+    @exhibits << exhibit
+  end
+
+  def recommend_exhibits(patron)
+    @exhibits.select do |exhibit| 
+      patron.interests.include?(exhibit.name)
+    end 
+  end
+
+  def admit(patron)
+    @patrons << patron
+  end
+
+  def patrons_by_exhibit_interest
```

Comment:
Yes!! This is exactly what I would do for this method, using only .each and if statements!  I also love that you are really digging in to using pry - it is such a good debugging tool :)